node/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = '2018'
 license = 'Apache-2.0'
 repository = 'https://github.com/digicatapult/vitalam-node/'
 name = 'vitalam-node'
-version = '2.9.3'
+version = '3.0.0'
```

Comment:
Why are we major version bumping the node?

Reply:
My bad, I'll change it to 10

Reply:
2.10.0